pkg/keccak/Makefile.include

```diff
@@ -0,0 +1,19 @@
+INCLUDES += -I$(PKGDIRBASE)/keccak/
+
+INCLUDES += -I$(PKGDIRBASE)/keccak/Common
```

Comment:
the includes need to be sorted out. I don't think that Keccac800 users need 16 global include paths. Also, "OptimizedAsmArm" sounds very arm specific.

Reply:
This was a remainder from the early days of experimenting, I've cleaned up this file and reduced it to seven includes. I will push it together with the changes to the Makefile.